distributed_index_tests/lib/user.py

```diff
@@ -76,7 +112,7 @@ def update_doc(self, doc_id, num_revision=1):
                 if put_resp.status_code == 201:
                     data = put_resp.json()
                     if data["rev"]:
-                        self.docs_info[doc_id] = data["rev"]  # init doc revisions to 0
+                        self.cache[doc_id] = data["rev"]  # init doc revisions to 0
```

Comment:
I though about it a bit and I agree it is not perfect, but docs_info seemed more ambiguous especially when comparing with changes. We would have things like changes_doc_ids compared with doc_info_ids. I thought too many *_docs_* may get confusing and could be easier to make mistakes when validating. Do you have any name suggestions?